relay-dynamic-config/Cargo.toml

```diff
@@ -30,9 +30,6 @@ relay-quotas = { workspace = true }
 relay-sampling = { workspace = true }
 serde = { workspace = true }
 serde_json = { workspace = true }
-smallvec = { workspace = true }
 
 [dev-dependencies]
-insta = { workspace = true }
```

Comment:
this is probably required

Reply:
for `cargo insta review`

Reply:
I couldn't find any usages inside `relay-dynamic-config`. Is this line still required to run `cargo insta review` even if the workspace has `insta` as a dependency?

Reply:
`cargo insta` commands require the crate installed on the host, so it doesn't have to be in dependencies.